src/sql/workbench/browser/modelComponents/inputbox.component.ts

```diff
@@ -141,7 +142,19 @@ export default class InputBoxComponent extends ComponentBase implements ICompone
 
 	public validate(): Thenable<boolean> {
 		return super.validate().then(valid => {
-			this.inputElement.validate();
+			const inputElemValid = this.inputElement.validate();
```

Comment:
Wouldn't `inputElemValid` need to be combined with `valid` in order to becomethe new output of this function?  Something like `valid = valid && this.inputElement.validate();` then just switch on and return `valid` below.

Reply:
Makes sense to me. However one thing I haven't been able to pinpoint is if the secondary validate check is redundant. In my original version of this fix, I didn't make the secondary check and everything worked as expected.